src/hotspot/share/opto/superword.hpp

```diff
@@ -495,6 +495,11 @@ class SuperWord : public ResourceObj {
     return _vloop_analyzer.dependency_graph().mutually_independent(nodes);
   }
 
+  // VLoopDependencyGraph Accessors
+  const VPointer& get_pointer(const MemNode* mem) const {
```

Comment:
This can't just be called `vpointer` or `vpointer_of`?

Reply:
+1 for `vpointer()`

Reply:
Good idea!